src/witan/app/schema.clj

```diff
@@ -44,9 +44,9 @@
 (def ModelPropertyType
   "Types that the model properties can be"
   (s/enum
-   "text"
-   "number"
-   "dropdown"))
+   :text
+   :number
+   :dropdown))
```

Comment:
for this and the util note: I found that 'undo', when used ineptly, is particularly dangerous in a rebase context